src/themes/cobalt.ts

```diff
@@ -795,13 +795,17 @@ export const cobaltTheme = {
     borderLeft: '',
     borderRight: `1px solid ${cobaltColors.lt800}`,
     transition: 'min-width 0.15s',
+    label: {
+      color: cobaltColors.drk800,
+    },
+
     secondaryNavbar: {
       width: '0px',
       openWidth: '170px',
       background: cobaltColors.white,
-      padding: '80px 0 16px',
+      padding: '30px 0 16px',
       margin: '',
-      zIndex: 4,
+      zIndex: 100,
```

Comment:
Instead of this zIndex can we not just use the one directly under sidenav? Since both are 100